spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/java/com/example/SecretManagerWebController.java

```diff
@@ -62,6 +63,10 @@ public class SecretManagerWebController {
   @Value("${sm@application-secret:DEFAULT}")
   private String appSecretFromValue;
 
+  // Application secrets with regions can be accessed with this syntax.
+  @Value("${sm@locations/us-central1/application-secret:DEFAULT}")
```

Comment:
Can you add step to [README](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/README.adoc) on setting up this regional secret (after step 4)?

Also, I am not familiar with this, but can global and regional secret co-exist in same project?  From [this page](https://cloud.google.com/secret-manager/regional-secrets/global-regionalized-service-comparison), it sounds like a choice per org?